Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed Error logged when InputActionEditor window opened without a valid asset.
 - Fixed ArgumentNullExceptions thrown when deleting items quickly in the UITK Editor.
 - Fixed Project Settings header title styling for Input Actions editor.
+- Fixed Input Actions Editor losing reference to current ControlScheme upon entering PlayMode [ISXB-770](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-770).
```

Comment:
Nitpick but think it should be "Play Mode" since that is how its referred to in docs?